crates/wasmi/src/store.rs

```diff
@@ -29,6 +28,8 @@ use crate::{
     TableEntity,
     TableIdx,
 };
+#[cfg(feature = "std")]
```

Comment:
With `use alloc::boxed::Box` none of the `#[cfg(feature = "std")]` is needed since this import works both in `std` and `no_std` environments.

Reply:
got it. fixed.